tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -402,6 +343,74 @@ public:
    std::size_t EstimateWriteMemoryUsage(const RNTupleWriteOptions &options = RNTupleWriteOptions()) const;
 };
 
+namespace Internal {
```

Comment:
Why move the classes? This makes it kind of difficult to compare the changes you've made inside these classes..

Reply:
Because otherwise it cannot compile due to needing the fwd-decl in one class but also the complete type in the other